### ISSUE TITLE
update council district map

### ISSUE THREAD

**Issue by chaserx** [openlexington/whats_my_district_redux#25]
https://twitter.com/CDoerge/status/597005456557629440

**Comment by chaserx** [comment]
http://data.lexingtonky.gov/dataset/board-of-elections-council-district

**Comment by chaserx** [comment]
trouble converting to geojson:

```
chase at foxy in ~/Downloads/Council  as CS using 2.2.1
○ ogr2ogr -f GeoJSON -t_srs crs:84 council.geojson Council.shp
Can't transform coordinates, source layer has no
coordinate system.  Use -s_srs to set one.
ERROR 1: Terminating translation prematurely after failed
translation of layer Council (use -skipfailures to skip errors)
```

**Comment by chaserx** [comment]
fixed by #26
